_posts/2018-07-13-FR-team.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Programming Historian French initiative 
+authors:
+- Anna-Maria Sichani
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/avatars/Marie-Puren.png" alt=""/><figcaption>
```

Comment:
Fix the nesting of these tags please. Both `<img>` elements can sit inside one `<figure>` element.